docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -63,53 +62,101 @@ spec:
+      name: multi-version-konflux-sample-{{.versionName}}
     spec:
-      application: "konflux-multibranch-sample-{{.versionName}}"
-      componentName: "konflux-multibranch-sample-{{.versionName}}"
+      application: "multi-version-konflux-sample-{{.versionName}}"
+      componentName: "multi-version-konflux-sample-{{.versionName}}"
       source:
         git:
           context: ./
           dockerfileUrl: Dockerfile
           revision: "{{.version}}"
-          url: https://github.com/ifireball/konflux-multibranch-sample.git
+          url: https://github.com/konflux-ci/multi-version-konflux-sample.git
+
+  - apiVersion: appstudio.redhat.com/v1alpha1
+    kind: ImageRepository
+    metadata:
+      name: imagerepository-for-multi-version-konflux-sample-{{.versionName}}
+      labels:
+        appstudio.redhat.com/application: multi-version-konflux-sample-{{.versionName}}
+        appstudio.redhat.com/component: multi-version-konflux-sample-{{.versionName}}
+    spec:
+      image:
+        name: konflux-samples-tenant/multi-version-konflux-sample-{{.versionName}}/multi-version-konflux-sample-{{.versionName}}
+        visibility: public
+      notifications:
+        - config:
+            url: 'https://bombino.preprod.api.redhat.com/v1/sbom/quay/push'
+          event: repo_push
+          method: webhook
+          title: SBOM-event-to-Bombino
+
 ----
 
 The *resources* section for the template may be created by looking at the YAML for existing resources and copying it while removing generated and unnecessary data and adding variable references where needed.
 
+The following kinds of resources are currentyl supported. Please do not attempt to create other kinds as that would prevent the template from being applied successfully.
+
+[cols="2,1,3"]
+|===
+|apiVersion |kind |Comments
+
+|appstudio.redhat.com/v1alpha1
+|Application
+|
+
+|appstudio.redhat.com/v1alpha1
+|Component
+|
+
+|appstudio.redhat.com/v1alpha1
+|ImageRepository
+|Every component should have an accompanying `ImageRepository` resource
+
+|appstudio.redhat.com/v1beta2
+|IntegrationTestScenario
+|Note the `v1beta2` API version. Make sure to use the right version when querying the cluster, otherwise important data may be missing from the resutls you get.
+
+|appstudio.redhat.com/v1alpha1
+|ReleasePlan
+|
+|===
+
+*Note:* When in doubt - the source of truth about what the YAML for your resources should look like is the up to date YAML you can find on the Konflux clusters. We try to keep the exaples here up to date, but with this being a static document there may be a gap between what you see here and what you should configure in tour system. It is recommended that you follow the process of creating the template by copying the YAML of the resrouces you have and cleaning it as describe blow, and not by copying the sample you see here.
```

Comment:
```suggestion
*Note:* When in doubt - the source of truth about what the YAML for your resources should look like is the version you can find on the {ProductName} clusters. We try to keep the examples here up to date, but with this being a static document there may be a gap between what you see here and what you should configure in your system. It is recommended that you follow the process of creating the template by copying the YAML of the resources you have and cleaning it as describe below, and not by copying the sample you see here.
```

What are you referring to specifically here? Just the resources that are mentioned above? Maybe we can reword this a little bit to say something like this?
```suggestion
*Note*: While we will try to keep the samples above current, they are a snapshot in time. In order to ensure that you are initializing the Project Controller with the latest and up to date versions of the resources, you should follow the process below for preparing the controller's resources.
```

Reply:
The template resource includes copeis of other resources with template variable references added in various places. I know that people tend to copy the example here and then try to modify it for their needs. This can end up trying to create resources with outdated settings. I'm trying here to encourage people to create the template by copying the resources they find in the cluster instead.

Your 2nd suggestion does not seem to me to point at the right direction, accepting the first.